unit-tests/src/test/java/org/eclipse/collections/impl/set/mutable/primitive/AbstractByteSetTestCase.java

```diff
@@ -388,4 +388,14 @@ public void classIsNonInstantiable()
     {
         Verify.assertClassNonInstantiable(ByteSets.class);
     }
+
+    @Test
+    public void union()
```

Comment:
I think we should add tests for both sets being empty, and for either being empty.

Reply:
Added tests for 1. both sets being empty. 2. left (set1) is empty 3. right (set2) is empty.
Lines: 410-423.